src/lib_json/json_value.cpp

```diff
@@ -190,6 +191,7 @@ static inline void releaseStringValue(char* value, unsigned) {
 
 namespace Json {
 
+#if JSON_USE_EXCEPTION
```

Comment:
I don't understand the need for this guard. As long as we don't *throw* an exception, who cares if exceptions are define? Also, I think it's still declared, right?